app/integration_test/README.md

```diff
@@ -1,211 +1,53 @@
-══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞═════════════════
-The following TestFailure object was thrown running a test:
-  Expected: <5>
-  Actual: <4>
-
-When the exception was thrown, this was the stack:
-#4      main.<anonymous closure> (file:///Users/jonassharezone/development/projects/sharezone-app/app/integration_test/foo_test.dart:8:5)
-#5      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:144:29)
-[Ausgelassen für weniger Text]
-#35     Invoker._onRun.<anonymous closure> (package:test_api/src/backend/invoker.dart:369:7)
-#42     Invoker._onRun (package:test_api/src/backend/invoker.dart:368:11)
-#43     LiveTestController.run (package:test_api/src/backend/live_test_controller.dart:153:11)
-(elided 31 frames from dart:async and package:stack_trace)
-
-This was caught by the test expectation on the following line:
-  file:///Users/jonassharezone/development/projects/sharezone-app/app/integration_test/foo_test.dart line 8
-The test description was:
-  failing test example
-═════════════════════════════════════════════════════════════════
-
-end of failure 1
-
-
-
-Stopping application instance.
-Driver tests failed: 1
+### Mobile
+You can run the integration tests using the `flutter drive` command:
+```sh
+flutter test \
+  integration_test \
+  --flavor dev \
+  --dart-define \
+  USER_1_EMAIL="YOUR@EMAIL.COM" \
+  --dart-define \
+  USER_1_PASSWORD="YOUR_PASSWORD"
 ```
-</details>
 
 ### Web
-Der Test kann wie folgt auf dem Web gelaufen lassen werden:
-1. Falls nicht bereits vorhanden `chromedriver` installieren. (z.B. per `brew install chromedriver`)
-2. `chromedriver --port=4444` ausführen und einen Server starten
-3. In einem anderem Terminal aus dem `app`-Ordner den unten stehenden Flutter-Befehl ausführen
-```
-flutter drive \
-  --driver=test_driver/integration_test.dart \
-  --target=integration_test/foo_test.dart \
+_Note: The integration tests are not working for the web at the moment because we need to migrate our app to null safety. Otherwise, the build will fail because of this message:_
+```
+org-dartlang-app:/app_test.dart:27:11: Error: Non-nullable variable 'dependencies' must be assigned before it can be used.
+    await dependencies.blocDependencies.auth.signOut();
+          ^^^^^^^^^^^^
+org-dartlang-app:/app_test.dart:34:30: Error: Non-nullable variable 'dependencies' must be assigned before it can be used.
+          beitrittsversuche: dependencies.beitrittsversuche,
+                             ^^^^^^^^^^^^
+org-dartlang-app:/app_test.dart:35:29: Error: Non-nullable variable 'dependencies' must be assigned before it can be used.
+          blocDependencies: dependencies.blocDependencies,
+                            ^^^^^^^^^^^^
+org-dartlang-app:/app_test.dart:36:28: Error: Non-nullable variable 'dependencies' must be assigned before it can be used.
+          dynamicLinkBloc: dependencies.dynamicLinkBloc,
+                           ^^^^^^^^^^^^
+```
```

Comment:
It seems like this might happen because on web it runs with null-safety? 

Couldn't we either:    

1. Use `late` for group variables
```dart
  late AppDependencies dependencies;
  late _UserCredentials user1;
```

2. Pass `--no-sound-null-safety` explicitly

3. Make the `app_test` file use null-safety by using `// @dart=2.14`

(I haven't tried any of it - just general ideas.)

Reply:
I also tried this, but you will get a different error other packages require null safety.